### DISCUSSION TITLE
Is LinkML mapped to ISO 11179-3?

### DISCUSSION THREAD

**Discussion by diatomsRcool** [linkml/linkml#2197] [Q&A] [answered]
Is LinkML mapped to ISO 11179-3?

**Comment by cmungall** [comment, marked as answer]
Good Q!

The LinkML metamodel is strongly influenced by and attempts to conform to the model in ISO/IEC 11179-3 - Metadata registries -- Part 3: [Registry metamodel and basic attributes](https://www.iso.org/standard/78915.html).

AFAIK there are not specific PURLs for the different elements of ISO-11179-3, but as a rough sketch:

* Data Elements (DEs) typically map to *slots*/*attributes*
* The Data Element Concept (DEC) is indicated using the slot_uri and class_uri or the containing class
* The Value Domain (ED) is the *range* of the slot, which will typically be an enum or a type
    * Enumerated Value Domains (EVDs) map to LinkML enums, and Permissible Value (PV) to LinkML PermissibleValue
        * Value Meaning (VM) are represented using `meaning` in LinkML
    * Described Value Domains (DVDs) (e.g. string for a description, bool for Y/N, measurement values) to LinkML types

Here are some early sides from the LinkML drive illustrating the mapping: 

https://docs.google.com/presentation/d/1w0FTBMMNldG9Juh1f9D1M7y13Jy8DdE76dDWOaMwK2Y/edit#slide=id.gb6cebe2b7f_0_0
